specification/security/resource-manager/Microsoft.Security/stable/2020-01-01/assessmentMetadata.json

```diff
@@ -532,7 +531,11 @@
           "description": "Secret to authenticate the partner and verify it created the assessment - write only",
           "x-ms-secret": true
         }
-      }
+      },
+      "required": [
+        "partnerName",
+        "secret"
+      ]
```

Comment:
Similar breaking change as the other comment.

Reply:
same answer :)

Reply:
Sounds good.